peer/core.yaml

```diff
@@ -314,11 +314,15 @@ chaincode:
     #net - in net mode validator will run chaincode in a docker container
 
     mode: net
+
     # typically installpath should not be modified. Otherwise, user must ensure
     # the chaincode executable is placed in the path specifed by installpath in
     # the image
     installpath: /opt/gopath/bin/
 
+    # The NetworkMode in HostConfig when starting a chaincode container
```

Comment:
Could you add some documentation here about the available options and why someone may wish to change this setting?